src/index.tsx

```diff
@@ -160,6 +160,32 @@ function loadResources() {
 }
 loadResources();
 
+/**
+ * Set's up Monaco Editor's Workers.
+ */
+enum Workers {
+  Json = 'json',
+  Editor = 'editor',
+}
+
+(window as any).MonacoEnvironment = {
```

Comment:
```suggestion
import { Environment } from 'monaco-editor/esm/vs/editor/editor.api';
// ... other imports

declare global {
  interface Window {
    MonacoEnvironment: Environment;
  }
}

window.MonacoEnvironment = {
```

To reduce using `any`s, assign the `MonacoEnvironment` on the `window`.